app/views/newflow/login_signup/educator_signup_form.html.erb

```diff
@@ -133,15 +145,20 @@
 
         <div class="content">
           <%= f.submit I18n.t(:"login_signup_form.continue_button"),
-                       data: { disable_with: I18n.t(:"login_signup_form.continue_button") },
+                       data: { disable_with: false },
                        class: 'primary', id: 'signup_form_submit_button',
                        :data => {ga_category: 'Account Creation', ga_action: 'Click', ga_label: '3A-Email'} %>
         </div>
       <% end %>
     <% end %>
   </div>
+
+  <% content_for :page_specific do %>
+    <%= javascript_include_tag 'newflow/signup_email_validations' %>
```

Comment:
Very nice how you did this, btw, creating page-specific JS 🙌 . Feel free to rename this `newflow/signup_email_validations` file. I didn't give it a second thought then, but at least right now — these email validations are only for educators, so I think I should've called it `educator_signup_email_validations`.